src/onepay.js

```diff
@@ -0,0 +1,67 @@
+const OnepayDirectQr = require('./onepay-direct-qr');
+const OnepayCheckout = require('./onepay-checkout');
+const SmartPhone = require('./smartphone');
+
+class Onepay {
+  constructor(transaction) {
+    if (transaction) {
+      this.transaction = transaction;
+    }
+  }
+
+  static directQr(transaction, htmlTagId) {
+    let onepay = new OnepayDirectQr(transaction);
+    onepay.drawQrImage(htmlTagId);
+  }
+
+  static checkout(options) {
+    let checkout = new OnepayCheckout(options);
+    checkout.pay();
+  }
+
+  drawQrImage(htmlTagId) {
+    let onepay = new OnepayDirectQr(this.transaction);
+    onepay.drawQrImage(htmlTagId);
+  }
+}
+
+SmartPhone.userAgent = null;
+
+if (typeof window === 'function' || typeof window === 'object') {
+  SmartPhone.setUserAgent(navigator.userAgent);
+}
+
+if (typeof exports !== 'undefined') {
```

Comment:
It seems like this is for checking if the code is being used on node.js (https://stackoverflow.com/questions/12622895/what-is-exports-object-for)

Reply:
@leosoto @RodrigoAyala @afiebig please your opinions about this wil be helpfull in order to merge this PR asap.